zone-service/src/main/java/org/cloudfoundry/identity/uaa/zone/OrchestratorZoneService.java

```diff
@@ -114,11 +117,9 @@ public void setDefaultSamlSignatureAlgorithm(@Qualifier("globalSamlSignatureAlgo
 
     public OrchestratorZoneResponse getZoneDetails(String zoneName) {
         OrchestratorZoneEntity orchestratorZone = zoneProvisioning.retrieveByName(zoneName);
-        OrchestratorZone zone = new OrchestratorZone(null, getSubDomainStr(orchestratorZone));
-        String uaaUri = ServletUriComponentsBuilder.fromCurrentContextPath().toUriString();
         String subDomain = orchestratorZone.getSubdomain();
-        String zoneUri = getZoneUri(subDomain, uaaUri);
-        ConnectionDetails connectionDetails = buildConnectionDetails(zoneName, orchestratorZone, zoneUri);
+        String zoneUri = getZoneUri(subDomain);
+        ConnectionDetails connectionDetails = buildConnectionDetails(orchestratorZone, zoneUri);
```

Comment:
Move lines 120-121 inside buildConnectionDetails() that can take just one parameter - orchestratorZone

Reply:
Done. Thanks!